docs/reference/storage-reference.md

```diff
@@ -1,5 +1,109 @@
+
+<h5> Example </h5>
+
+```python
+storage.get_assets(
+    created_after="2021-01-01",
+    created_before="2023-01-01",
+    workspace_id="68567134-27ad-7bd7-4b65-d61adb11fc78",
+    collection_names=["phr"],
+    producer_names=["airbus"],
+    tags=["optical", "us"],
+    sources=["ARCHIVE"],
+    search=["NY Central Park"],
+    limit=5,
+    sortby="productId",
+    return_json=True,
+)
+```
+
+## Orders
+
+### get_orders()
+
+The `get_orders()` function allows you to search for tasking and catalog orders in the workspace.
+
+```python
+get_orders(
+    return_json,
+    limit,
+    sortby,
+    descending,
+    order_type,
+    tags,
+)
+```
+
+The returned format is `Union[list[Order], dict]`.
+
+<h5> Arguments </h5>
+
+| Argument      | Overview                                                                                                                                                                                                                                                |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `return_json` | **bool**<br/>Determines how to return the orders.<br/><ul><li>`True`: returns a JSON dictionary</li><li>`False`: returns a list of order objects.</li></ul>The default value is `False`.                                                                |
+| `limit`       | **int**<br/>The number of assets in an order to return.                                                                                                                                                                                                 |
+| `sortby`      | **str**<br/>The results sorting method that arranges the returned orders. The allowed values:<br/><ul><li>`"createdAt"`</li><li>`"updatedAt"`</li><li>`"status"`</li><li>`"dataProvider"`</li><li>`"type"`</li></ul>The default value is `"createdAt"`. |
+| `descending`  | **bool**<br/>Determines the sorting order of the returned orders.<br/><ul><li>`True`: descending</li><li>`False`: ascending</li></ul>The default value is `True`.                                                                                       |
+| `order_type`  | **str**<br/>The type of orders to return. The allowed values:<br/><ul><li>`"TASKING"`: use to get only tasking orders</li><li>`"ARCHIVE"`: use to get only catalog orders</li></ul>Don't use the parameter to get orders of all types.                  |
```

Comment:
Remove `"`

Reply:
"Don't use the parameter to get orders of all types." (I see that you took it from the reference, but I think there's a better way to describe it) -> To get orders of all types, omit the parameter.

Reply:
There's something wrong with line spacing above the sentence

Reply:
After much investigation, I think the line spacing is like this because the sentence `To get orders of all types, omit the parameter.` doesn't have any text within the back ticks, like how the others have the default value written `True`. 
In my commit, I can add an empty ` ` at the end of the sentence, and it will add the line spacing. But moving this sentence to the top of the description or omitting the sentence entirely would solve the problem, too. 
What is your preference? @dustgalactic

Reply:
Let's move it to the top :)